src/Illuminate/Foundation/Vite.php

```diff
@@ -37,6 +38,27 @@ class Vite
      */
     protected $styleTagAttributesResolvers = [];
 
+    /**
+     * The registered entry points.
+     *
+     * @var array
+     */
+    protected $entryPoints = [];
+
+    /**
+     * The path to the hot file.
+     *
+     * @var string|null
+     */
+    protected $hotFilePath;
+
+    /**
+     * The path to the build directory.
+     *
+     * @var string
+     */
+    protected $buildDirectory = 'build';
```

Comment:
@timacdonald Should this value be empty as the `hotFilePath` by default? The `__invoke` method will handle the rest.

Reply:
As it is, this could make sense, but with the tweaks I've made and a recent feature I've added I think it now makes sense to leave this in place.